dcos/cluster.py

```diff
@@ -391,11 +391,11 @@ def get_dcos_version(self):
                 url=endpoint,
                 timeout=5,
                 verify=False)
+
+            return resp.json().get("version", VERSION_UNKNOWN)
```

Comment:
would that be appropriate here to try catch jsonification here?

Reply:
what do you mean? it is currently in a catch-all block.